COLLADAMaya/src/COLLADAMayaAnimationExporter.cpp

```diff
@@ -1004,6 +1022,82 @@ namespace COLLADAMaya
         }
     }
 
+
+	const String AnimationExporter::getNameOfStepInterpolation(const Step & type)
+	{
+		String FinalStep = "";
+
+		
+
+		if ((type._transform & 0x20) >> 5)
+		{
+			String step = type._type[5] == STEPPED ? "STEP" : "STEPNEXT";
+			FinalStep += String((FinalStep.size() != 0 ? "|" : "") + step + String("_RZ"));
+		}
+		if ((type._transform & 0x10) >> 4)
+		{
+			String step = type._type[4] == STEPPED ? "STEP" : "STEPNEXT";
+			FinalStep += String((FinalStep.size() != 0 ? "|" : "") + step + String("_RY"));
+		}
+		if ((type._transform & 0x8) >> 3)
+		{
+			String step = type._type[3] == STEPPED ? "STEP" : "STEPNEXT";
+			FinalStep += String((FinalStep.size() != 0 ? "|" : "") + step + String("_RX"));
+		}
+		if ((type._transform & 0x4) >> 2)
+		{
+			String step = type._type[2] == STEPPED ? "STEP" : "STEPNEXT";
+			FinalStep += String((FinalStep.size() != 0 ? "|" : "") + step + String("_TZ"));
+		}
+		if ((type._transform & 0x2) >> 1)
+		{
+			String step = type._type[1] == STEPPED ? "STEP" : "STEPNEXT";
+			FinalStep += String((FinalStep.size() != 0 ? "|" : "") + step + String("_TY"));
+		}
+		if ((type._transform & 0x1) >> 0)
+		{
+			String step = type._type[0] == STEPPED ? "STEP" : "STEPNEXT";
+			FinalStep += String((FinalStep.size() != 0 ? "|" : "") + step + String("_TX"));
+		}
+
+		return FinalStep;
+	}
+
+	const String AnimationExporter::getNameOfOrder(const MEulerRotation::RotationOrder & order)
+	{
+		String FinalOrder;
+
+		switch (order)
+		{
+			case MEulerRotation::kXYZ:
+				FinalOrder = "ZYX";
+				break;
+			case MEulerRotation::kXZY:
+				FinalOrder = "YZX";
+				break;
+			case MEulerRotation::kYXZ:
+				FinalOrder = "ZXY"; 
+				break;
+			case MEulerRotation::kYZX:
+				FinalOrder = "XZY";
+				break;
+			case MEulerRotation::kZXY:
+				FinalOrder = "YXZ";
+				break;
+			case MEulerRotation::kZYX:
+				FinalOrder = "XYZ";
+				break;
+			default:
+				// Export XYZ euler rotation in Z Y X order in the file.
+				// The rotation order is set to XYZ, Collada reads the parameter from behind.
```

Comment:
This is exactly done in this order in ColladaMayaRotateHelper.cpp

```javascript
 void RotateHelper::createRotation ( )
    {
        switch (rotation.order)
        {
        case MEulerRotation::kXYZ: 
            createZ(0); createY(1); createX(2); 
            break;

    ...
}
```

This code will add a new technique into source channel.

```javascript
 <technique profile="OpenCOLLADAMaya">
          <step>STEP_RZ|STEP_RY|STEP_RX|STEP_TZ|STEP_TY|STEPNEXT_TX STEP_RZ|STEP_RY|STEP_RX|STEP_TZ|STEP_TY|STEP_TX </step>
          <order>ZYX </order>
        </technique>
```